aioshelly/coap.py

```diff
@@ -102,10 +111,12 @@ def datagram_received(self, data, addr):
             self._message_received(msg)
 
         if msg.ip in self.subscriptions:
+            _LOGGER.debug("Adding CoAP message to subscriptions for device %s", msg.ip)
```

Comment:
```suggestion
            _LOGGER.debug("Calling CoAP message update for device %s", msg.ip)
```
This is not adding the message, but it is calling the update handler subscribed at `subscribe_updates`